dashboard.css

```diff
@@ -0,0 +1,325 @@
+body {
```

Comment:
Can you please highlight or put comments on where we did customisation compared to the file that got copied?

Reply:
sure, will update

Reply:
Have added comments in css with the format as
 _"edit: Adding default tile color (defining reason for change)"_